app/views/faq.scala.html

```diff
@@ -337,39 +340,39 @@ <h3 class="govuk-heading-s">
                 </p>
                 <ul class="govuk-list govuk-list--bullet">
                     <li>Redact - remove the sensitive words. For example: 'Correspondence with [name withheld]'.</li>
-                    <li>Sanitise - replace the sensitive word. For example: 'UK military exercises in xx' (where 'military' has replaced 'Special Forces').</li>
-                    <li>Withhold the title or description without providing an alternative. For example: '[Title withheld]' and or '[description withheld]'</li>
+                    <li>Sanitise - replace the sensitive word. For example: 'UK military exercises in...' (where 'military' has replaced 'Special Forces').</li>
+                    <li>Withhold the title or description without providing an alternative. For example, you can replace with '[Title withheld]' and or '[description withheld]'</li>
                 </ul>
 
                 <h3 class="govuk-heading-s">Can I include Redacted records?</h3>
                 <p class="govuk-body">
                     If redacted files are included within your transfer, to avoid failing checks, you must ensure that:
                 </p>
                 <ul class="govuk-list govuk-list--bullet">
-                    <li>They are in the same subfolder as their parent document</li>
+                    <li>They are in the same subfolder as their parent record</li>
```

Comment:
Think "subfolder" should be hypenated as per other instances: `... same sub-folder as their ...`